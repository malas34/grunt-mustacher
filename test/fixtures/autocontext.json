{
    "title": "Titre de la page autocontext",
    "content": "Contenu depuis le data autocontext"
}
