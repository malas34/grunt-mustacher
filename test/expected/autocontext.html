<html>
    <head>
        <title>Titre de la page autocontext</title>
    </head>
    <body>Contenu depuis le data autocontext</body>
</html>
